rpm/kubelet.spec

```diff
@@ -26,14 +26,13 @@ Source1: kubelet.service
 Source2: https://dl.k8s.io/v%{KUBE_VERSION}/bin/linux/%{ARCH}/kubectl
 Source3: https://dl.k8s.io/v%{KUBE_VERSION}/bin/linux/%{ARCH}/kubeadm
 Source4: 10-kubeadm.conf
-Source5: https://dl.k8s.io/network-plugins/cni-plugins-%{ARCH}-v%{CNI_VERSION}.tgz
+Source5: https://storage.googleapis.com/k8s-artifacts-cni/release/v%{CNI_VERSION}/cni-plugins-linux-%{ARCH}-v%{CNI_VERSION}.tgz
 Source6: kubelet.env
 Source7: https://github.com/kubernetes-sigs/cri-tools/releases/download/v%{CRI_TOOLS_VERSION}/crictl-v%{CRI_TOOLS_VERSION}-linux-%{ARCH}.tar.gz
 
 BuildRequires: systemd
 BuildRequires: curl
 Requires: iptables >= 1.4.21
-Requires: kubernetes-cni >= %{CNI_VERSION}
```

Comment:
This should probably change from Requires: to Obsoletes:, eg:

Obsoletes: kubernetes-cni

I don't know the debian meta options.  Is there something like this there too?

Reply:
Thanks Tim! I'm going to do a little research and come back with the deb equivalent as well next week.

Reply:
@tpepper -- This should be ready to merge now. See the worklog below: https://github.com/kubernetes/release/pull/1309#issuecomment-636434847

I ended up adding this to the rpm spec:

```
Obsoletes: kubernetes-cni
Conflicts: kubernetes-cni
```

and this to the debian package config:

```
Provides: kubernetes-cni
Conflicts: kubernetes-cni
Replaces: kubernetes-cni
```